cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/purgatory/RequestInfo.java

```diff
@@ -33,6 +33,8 @@
   static {
     VALID_TRANSFER.put(PENDING_REVIEW, new HashSet<>(Arrays.asList(APPROVED, DISCARDED)));
     VALID_TRANSFER.put(APPROVED, new HashSet<>(Arrays.asList(DISCARDED, SUBMITTED)));
+    VALID_TRANSFER.put(SUBMITTED, new HashSet<>());
+    VALID_TRANSFER.put(DISCARDED, new HashSet<>());
```

Comment:
Nit: For `SUBMITTED` and `DISCARDED`, can we make the `Collections.emptySet()` to save some bytes?